packages/hub/services/data-integrity-checks/cron-tasks.ts

```diff
@@ -0,0 +1,102 @@
+import { inject } from '@cardstack/di';
+import { Client } from 'pg';
+import { IntegrityCheckResult } from './utils';
+
+import config from 'config';
+
+export type TaskIdentifier =
+  | 'check-reward-roots'
+  | 'execute-scheduled-payments'
+  | 'remove-old-sent-notifications'
+  | 'print-queued-jobs';
+
+interface TaskConfig {
+  cronMinutes: number;
+}
+
+interface CronTask {
+  'execute-scheduled-payments': TaskConfig;
+  'check-reward-roots': TaskConfig;
+  'print-queued-jobs': TaskConfig;
+  'remove-old-sent-notifications': TaskConfig;
+}
+
+// graphile worker configuration (copied from worker-client)
```

Comment:
Is it possible to dynamically import this instead of copying?

Reply:
I think this is a good point. It did occur to me too but I thought to be conservative as to try not to touch the original code.

Let me investigate this.